components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from "next/link";
+import React from "react";
+import { Logo } from "../Logo/Logo";
+
+export interface Link {
+  label: string;
+  route: string;
+}
+
+interface FooterLinkProps extends React.HTMLProps<HTMLLIElement> {
+  route: string;
+}
+
+interface FooterProps extends Partial<React.HTMLProps<HTMLElement>> {
+  links: Link[];
+}
+
+const textClasses =
+  "font-outfit text-[0.875rem] text-center font-semibold text-white";
+
+const focusClasses = "focus:shadow-md focus:border-white focus:border-2";
+
+const disabledClasses = "disabled:cursor-not-allowed disabled:bg-gray";
+
+export const Footer = ({ links }: FooterProps) => {
+  return (
+    <footer
+      className="bg-blue pb-14 pt-10 md:pt-14"
+      aria-label="Footer"
+      data-testid="footer"
+    >
+      <nav className="mx-auto flex w-full max-w-[1040px] flex-col items-center justify-between md:flex-row">
+        <Link
+          className={`mx-2 rounded-sm border-2 border-transparent p-1 outline-none ${focusClasses}`}
```

Comment:
Can we move `mx-2` to the `div` or `nav`, if you will, that way both the logo and the text get this benefit

Reply:
Here wrapper has `mx-auto` already, I really don't wanna create an additional wrapper for that.

Reply:
Oh, I think I figured that out!

Reply:
Sorry!, I meant `px-2`